src/auth.js

```diff
@@ -45,14 +45,14 @@ export const askApiKey = () =>
  * Create the readers with apiKey if exist and Accept keys
  * @param {string?} accept Accept header, default is `application/json`
  */
-const getHeaders = (accept) => {
+const getHeaders = (accept = "application/json") => {
```

Comment:
why did you remove this?

Reply:
The functionality is the same. Default value is more transparent I think than https://github.com/prusa3d/Prusa-Link-Web/blob/master/src/auth.js#L52

Reply:
Got it. I misread it.